examples/visual-tests/amp-story/amp-story-page-attachment.html

```diff
@@ -49,23 +49,114 @@
         poster-landscape-src="/examples/visual-tests/picsum.photos/image981_1600x900.jpg"
         poster-square-src="/examples/visual-tests/picsum.photos/image981_1600x1600.jpg"
         supports-landscape>
-      <amp-story-page id="cover">
+      <amp-story-page id="inline-default">
         <amp-story-page-attachment layout="nodisplay">
           <h1>Some really cool video</h1>
           <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
           </p>
-          <amp-video width="auto" height="270" autoplay src="/examples/visual-tests/video/tokyo.mp4" controls>
```

Comment:
Why remove this?

Reply:
Since we're repeating the inline attachment 6 times, I thought I'd minimize it (thought it would be ok to truncate this since we're not testing video attachments here, but rather just the UI display overall)

But I agree w your comment below about leaving this file untouched and making a new test file for v2 UI. I'll do that and leave this file unchanged.